packages/flask/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -0,0 +1,220 @@
+                    filterCountries(r);
+                } else {
+                    setCountriesList(r);
+                }
+            })
+            .catch((e: Error) => {
+                console.error(e);
+            })
+            .finally(() => {
+                setLoading(false);
+            });
+    }, []);
+
+    const renderSearchHeader = () => {
+        return (
+            <View style={styles.headerContainer}>
+                <View style={styles.headerTitleContainer}>
+                    <View style={styles.sideHeaderView}>
+                        <UILinkButton onPress={onClose} title={uiLocalized.Cancel} />
+                    </View>
+                    <UILabel role={TypographyVariants.HeadlineHead} style={styles.headerTitle}>
+                        {uiLocalized.CountryPicker.Title}
+                    </UILabel>
+                    <View style={styles.sideHeaderView} />
+                </View>
+                <UISearchBar returnKeyType="done" value={search} onChangeText={setSearch} />
+            </View>
+        );
+    };
+
+    const renderCountryRow = ({ item }: { item: Country }) => {
+        const onPress = () => onSelectCountry(item);
+        return (
+            <TouchableOpacity onPress={onPress} style={styles.rowContainer}>
+                <View style={styles.rowContainerInner}>
+                    <UILabel>{item.name}</UILabel>
+                    <UILabel style={styles.emojiContainer}>{item.emoji}</UILabel>
+                </View>
+            </TouchableOpacity>
+        );
+    };
+
+    const renderLoading = () => {
+        return <ActivityIndicator />;
+    };
+
+    const renderEmptyList = () => {
+        return (
+            <>
+                <UILabel role={TypographyVariants.TitleMedium} color={ColorVariants.TextSecondary}>
+                    {uiLocalized.CountryPicker.CountryNotFindTitle}
+                </UILabel>
+                <UILabel role={TypographyVariants.Action} color={ColorVariants.TextSecondary}>
+                    {uiLocalized.CountryPicker.CountryNotFindDetails}
+                </UILabel>
+            </>
+        );
+    };
+
+    const ListEmptyComponent = React.useMemo(() => {
+        return (
+            <View style={styles.emptyContainer}>
+                {loading ? renderLoading() : renderEmptyList()}
+            </View>
+        );
+    }, [loading]);
+
+    const keyExtractor = React.useCallback((item: Country) => item.code, []);
+
+    return (
+        <UIBottomSheet onClose={onClose} visible={visible} style={styles.sheet}>
+            {renderSearchHeader()}
+            <FlatList
+                data={filteredList}
+                renderItem={renderCountryRow}
+                keyExtractor={keyExtractor}
+                ListEmptyComponent={ListEmptyComponent}
+                keyboardDismissMode="on-drag"
+            />
+        </UIBottomSheet>
+    );
+}
+
+const { height } = Dimensions.get('screen');
```

Comment:
Can you please use https://reactnative.dev/docs/0.64/usewindowdimensions instead? :)